sdk/provisioning/Azure.Provisioning.AppConfiguration/tests/BasicAppConfigurationTests.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Tests;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.AppConfiguration.Tests;
+
+public class BasicAppConfigurationTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+{
+    [RecordedTest]
+    [Description("https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.appconfiguration/app-configuration-store-ff/main.bicep")]
+    public async Task CreateAppConfigAndFeatureFlag()
+    {
+        await using Trycep test = CreateBicepTest();
+        await test.Define(
+            ctx =>
+            {
+                BicepParameter location = BicepParameter.Create<string>(nameof(location), BicepFunction.GetResourceGroup().Location);
```

Comment:
Why do we call all these types with the "Bicep" prefix? I think it would be ok if the feature is called Azure.Bicep, (not CDK). Then we just say "bicep" is the feature, not the file format.

Reply:
I waffled on this a ton and have renamed them five or six times.  I ultimately landed on BicepValue because they're values or Bicep expressions.  I'm very open to other name suggestions now that we've got more eyes here.  It's pretty cheap to change the entire API.  We just need to get that done sooner than later.